tests/network/BUILD.bazel

```diff
@@ -7,6 +7,7 @@ go_library(
         "expose.go",
         "framework.go",
         "networkpolicy.go",
+        "port-forward.go",
```

Comment:
please use underscore

Reply:
Done

Reply:
I think you renamed the file, but left the original file name in BUILD.bazel